### ISSUE TITLE
Cannot run grok after split filter with stdin and line codec workaround

### ISSUE THREAD

**Issue by beeblebrox** [elastic/logstash#2816]
This is similar to case #1820.  Opening a new case because the work around listed does not seem to work with me.

I'm using the stdin input. The input has newlines and I'm using split to emit multiple events based a different delimiter in the event ("/"). Afterwards I am attempting to use grok again and it does not work. Using the codec line for the input does not workaround the issue.
Config:
```
input {
  stdin { codec => "line" }
}
filter {
        split {
                terminator => "/"
        }
}
filter {
        grok {
                match => ["message", "(?<message_again>.*)"]
        }
}

output {
  stdout { codec => "rubydebug"}
}
```

Result:
```
this/test
Using milestone 2 filter plugin 'split'. This plugin should be stable, but if you see strange behavior, please let us know! For more information on plugin milestones, see http://logstash.net/docs/1.4.2/plugin-milestones {:level=>:warn}

{
       "message" => "this",
      "@version" => "1",
    "@timestamp" => "2015-03-13T03:54:17.944Z",
          "host" => "jhart"
}
{
       "message" => "test",
      "@version" => "1",
    "@timestamp" => "2015-03-13T03:54:17.944Z",
          "host" => "jhart"
}
```
Typing using a value without a split results in the correct message_again field.
Is there a workaround?

**Comment by jordansissel** [comment]
I think that the problems with split were fixed in 1.5; can you try 1.5.0 RC2? http://download.elasticsearch.org/logstash/logstash/logstash-1.5.0.rc2.zip

**Comment by beeblebrox** [comment]
I have confirmed it is working as expected in release 1.5.0 RC2.

This version might be fine for what I need this for. If not, is there a workaround for 1.4?  If not, I'm willing to patch 1.4 if you can point me to the commit which fixed this issue in 1.5.

**Comment by brupm** [comment]
@jordansissel is there a workaround?

**Comment by jordansissel** [comment]
Is this still an issue?

**Comment by jordansissel** [comment]
Looks like a prior comment reported this was fixed in 1.5.0RC2; so I will close.
